GVFS/GVFS.FunctionalTests/Tests/GitCommands/CheckoutTests.cs

```diff
@@ -498,6 +501,12 @@ public void CheckoutBranchWithOpenHandleBlockingRepoMetdataUpdate()
         [TestCase]
         public void CheckoutBranchWithOpenHandleBlockingProjectionDeleteAndRepoMetdataUpdate()
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                uint result = RegisterForOfflineIO();
+                Assert.Equals(PrjFSResultSuccess, result);

```

Comment:
Nit: We've been using the "Should" methods in the `.cs` tests, can you update this to use `ShouldEqual`:

`result.ShouldEqual(PrjFSResultSuccess, $"{nameof(RegisterForOfflineIO)} failed")`